pages/conduct/enforcement-manual.md

```diff
@@ -0,0 +1,60 @@
+<script>{
+	"title": "jQuery Foundation Code of Conduct - Enforcement Manual",
+	"pageTemplate": "page-conduct.php"
+}</script>
+
+This is the enforcement manual followed by jQuery's Code of Conduct Committee. It's used when we respond to an issue to make sure we're consistent and fair. It should be considered an internal document, but we're publishing it publicly in the interest of transparency.
```

Comment:
"...publishing it publicly..." is redundant. "publishing it" should be sufficient.